reledpar.dtx

```diff
@@ -804,16 +804,16 @@
 %   \end{Rightside}
 % \end{verbatim}
 % \subsection{Critical notes called on leftside but printed on right side}
-% Sometime, you need to print the critical notes on the right side, despite the refers to lemma of the leftside.
+% Sometimes, you need to print the critical notes on the right side, despite the references to the lemmas on the left side.
```

Comment:
> On 2016-07-01, at 03:47 , Maïeul Rouquette <notifications@github.com> wrote:
> 
> In reledpar.dtx <https://github.com/maieul/ledmac/pull/550#discussion_r69261607>:
> 
> > @@ -804,16 +804,16 @@
> >  %   \end{Rightside}
> >  % \end{verbatim}
> >  % \subsection{Critical notes called on leftside but printed on right side}
> > -% Sometime, you need to print the critical notes on the right side, despite the refers to lemma of the leftside.
> > +% Sometimes, you need to print the critical notes on the right side, despite the references to the lemmas on the left side.
> Sometimes, you need to print the critical notes on the right side, despite they refer to the lemmas on the left sid
> 
> 

Then write "Sometimes, you need to print the critical notes on the right side, despite the fact that they refer to the lemmas on the left side"


> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/maieul/ledmac/pull/550/files/abc94f75401202c18ccac19e1d798f85234d748f#r69261607>, or mute the thread <https://github.com/notifications/unsubscribe/AHG-QA_qnk2JV3TerNn-e0pi-K_jiF55ks5qRMX4gaJpZM4JC5Lv>.
>